e2e/test/iothub/service/QueryClientE2ETests.cs

```diff
@@ -213,9 +213,9 @@ public async Task JobQuery_QueryWorks()
             // tests in this suite are written to work even if the queried job isn't the one they created.
             // That's why these checks aren't more specific.
             queriedJob.JobId.Should().NotBeNull();
-            queriedJob.JobType.Should().NotBeNull();
+            queriedJob.JobType.Should().NotBe(JobType.Unknown);
```

Comment:
Enums can't be null by default, so we should simply compare against the default enum value.